src/pybind/mgr/rook/module.py

```diff
@@ -72,10 +74,13 @@ def __init__(self):
         self.namespace = os.environ.get('POD_NAMESPACE', 'rook-ceph')
 
         # ROOK_CEPH_CLUSTER_CRD_NAME is new is Rook 1.0
-        self.cluster_name = os.environ.get('ROOK_CEPH_CLUSTER_CRD_NAME', self.namespace)
+        self.cluster_name = os.environ.get(
+            'ROOK_CEPH_CLUSTER_CRD_NAME', self.namespace)
```

Comment:
could you move those style changes into a different commit?

Reply:
Yes. Sorry ... my IDE doing things by itself ... i will revert all these changes

Reply:
ping?